reader-gtfs/README.md

```diff
@@ -17,10 +17,10 @@ wget http://download.geofabrik.de/europe/germany/brandenburg-latest.osm.pbf
 # The following process will take roughly 5 minutes on a modern laptop when it is executed for the first time.
 # It imports the previously downloaded OSM data of the Brandenburg area as well as the GTFS.
 java -Xmx8g -Xms8g \
+# todonow: this is a) outdated, and b) why no CH (when in tests its enabled?)
```

Comment:
@michaz ? We probably want to update the command line parameters to `-Ddw.graphhopper.xyz` (since `-Dgraphhopper.xyz` is no longer working after #1879), and I was wondering why we are setting `ch.weightings=no` here (while for some gtfs tests we were using CH) (see other comment)

Reply:
Updadet the readme and removed `ch.weightings=no` as this is now the default